source/Operation Guides/InterFlux Masternodes/Using Masternode Setup Tool/masternode-setup-tool.rst

```diff
@@ -0,0 +1,309 @@
+************
+Introduction
+************
+
+This guide details the process required for joining the Cirrus
+Sidechain as a Block Producing Node, known as an InterFlux Masternode using the Masternode Setup Tool. The request to join the Cirrus Sidechain as a block
+producer, requires you to setup the Masternode Setup tool, therefore, a request to join the Cirrus Sidechain will be
+made once the Masternode is fully configured.
+
+Pre-Requisites
+==============
+
+To join as a InterFlux Masternode, there are a number of criteria
+that must be met.
+
+Wallet Requirements
+-------------------
+
+You must know the mnemonic and passphrase (if applicable) for a STRAX
+Wallet that contains the required collateral and fee.
+
+-  **Collateral Amount:** 100,000 STRAX
+
+-  **Fee Amount:** 501 STRAX
+
+
+Sidechain Masternode Hardware Requirements
+------------------------------------------
+
+The below hardware requirements are a minimum.
+
+-  2x vCPU / Dual Core
+
+-  8GB RAM
+
+-  50GB SSD / HDD
+
+Sidechain Masternode Software Requirements
+------------------------------------------
+
+There are only three pre-requisites software installations that need to
+take place before operating a InterFlux Masternode.
+
+.NET Core SDK
+~~~~~~~~~~~~~
+
+.NET Core is the platform that the InterFlux Masternode has been
+developed upon. Therefore, you will need to install the .NET Core SDK.
+
+Please install the latest version of .NET Core SDK 3.1
+
+https://dotnet.microsoft.com/download/dotnet-core/3.1
+
+Git for Windows
```

Comment:
We don't need Git to be installed on the target machine as we don't use it in MST (instead, we ship MST with it's own version of Stratis FullNode)